src/main/java/uk/gov/companieshouse/pscfiling/api/config/InterceptorConfig.java

```diff
@@ -80,7 +81,7 @@ private void addTransactionInterceptor(final InterceptorRegistry registry) {
 
     private void addOpenTransactionInterceptor(final InterceptorRegistry registry) {
         registry.addInterceptor(openTransactionInterceptor())
-                .addPathPatterns(COMMON_INTERCEPTOR_PATH).order(2);
+                .addPathPatterns(COMMON_INTERCEPTOR_RESOURCE_PATH).order(2);
```

Comment:
I can't think, but by modifying this to include the filings resource, does it mean any other requests are no longer going through this interceptor?